style.css

```diff
@@ -657,3 +669,683 @@ ul li {
   background-color: #fff;
   color: black;
 }
+
+@media (min-width: 768px) {
+  body {
+    height: 5125px;
+    width: 1440px;
+    left: 0;
+    top: 0;
+    border-radius: 0;
+    position: relative;
+    background: #dfe1e6;
+  }
+
+  .container {
+    position: relative;
+    width: 1440px;
+    height: 5125px;
+    background: #f4f5f7;
+  }
+
+  header {
+    position: absolute;
+    width: 1440px;
+    height: 92px;
+    left: 0;
+    top: 0;
+    padding-top: 40px;
+    border-radius: 0;
+    background-color: #fff;
+  }
+
+  .container header,
+  .main {
+    position: absolute;
+    width: 1440px;
+    height: 916px;
+    left: 0;
+    top: 1px;
+    background: #fff;
+    border-radius: 0 0 0 10%/0 0 0 10%;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .navbar-links {
+    position: absolute;
+    display: block;
+    padding: 0;
+    gap: 8px;
+    width: 258px;
+    height: 40px;
+    left: 1039px;
+    top: 16px;
+  }
+
+  .mylogo {
+    width: 76px;
+    height: 20px;
+    margin-left: 155px;
+    margin-top: -10px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .navbar-links ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 18px;
+    position: absolute;
+    width: 258px;
+    height: 40px;
+    left: 0;
+    top: 0;
+  }
+
+  .main {
+    background-image: url(img/Header\ bg@2x.png);
+    background-size: 100% 100%;
+    background-position: 100% 100%;
+    height: 721.4596557617188px;
+    width: 1397.3985595703125px;
+    left: 34.6015625px;
+    top: 119px;
+    border-radius: 0;
```

Comment:
As suggested by the previous reviewer, kindly remove the gray empty space at the right, your project is tilted to the left and this might be caused by using unresponsive values like `px` for width instead of `%`. please fix this before asking for  a re-review 😄 

your design :

https://user-images.githubusercontent.com/20537712/236647583-55ca1cb8-d083-467b-9911-281662e1454c.mp4

Figma design :
![image](https://user-images.githubusercontent.com/20537712/236647609-73b7c102-42ef-49b7-ab23-aa652dfea635.png)

Reply:
When I use inspect I don't see that Error it only shows up when using Zoom! @usorfaitheloho